scripts/lib/cluster.sh

```diff
@@ -96,6 +101,10 @@ function up-kops-cluster {
         let RETRY_ATTEMPT=RETRY_ATTEMPT+1
         echo "In attempt# $RETRY_ATTEMPT, waiting for cluster validation"
     done
+
+    kubectl delete daemonset ebs-csi-node -n kube-system
```

Comment:
nit: let's add a comment here explaining why these are being deleted

Reply:
Added comment here explaining the reason